node/main-extract-async.ts

```diff
@@ -181,7 +181,8 @@ export function startFileSystemWatching(
       const ext = filePath.substring(filePath.lastIndexOf('.') + 1);
 
       // WARNING - dangerously ignores any path that includes `vha-` anywhere!!!
-      if (filePath.indexOf('vha-') !== -1 || acceptableFiles.indexOf(ext) === -1) {
+      if (filePath.indexOf('vha-') !== -1 ||
+          (acceptableFiles.indexOf(ext) === -1 && GLOBALS.additionalExtensions.indexOf(ext) === -1)) {
```

Comment:
no need to change, just a thought: `...` spread operator could fuse the two arrays before `.indexOf()` 🤔